src/DataFixtures/ORM/LoadProgrammaticFoundationTagData.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace AppBundle\DataFixtures\ORM;
+
+use AppBundle\Entity\ProgrammaticFoundation\Tag;
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\Persistence\ObjectManager;
+
+class LoadProgrammaticFoundationTagData extends AbstractFixture
+{
+    public function load(ObjectManager $manager)
+    {
+        $tagLabels = [
```

Comment:
Can you make it a private constant instead of a local variable ? (like we do in other DataFixtures)

Reply:
Done !